### PR TITLE
access args by index

### PR DESCRIPTION
**Pull request by DeveloperInProgress**

### CODE REVIEW

**Review by stwiname — COMMENTED** [review]
This seems like a bug with the ethermint processor. Is this something we can fix there?

**Review by jamesbayly — APPROVED** [review]

**Comment by jamesbayly** [comment]
Agree that we need to fix something with the processor here

File: Cronos/cronos-evm-starter-via-rpc/src/mappings/mappingHandlers.ts [hunk 1]
```diff
@@ -20,12 +20,11 @@ type ApproveCallArgs = [string, BigNumber] & {
 export async function handleEthermintEvmEvent(
   event: EthermintEvmEvent<TransferEventArgs>
 ): Promise<void> {
-  logger.info(JSON.stringify(event.args.value));
   const transaction = Transaction.create({
     id: event.transactionHash,
-    value: event.args.value.toBigInt(),
```
Comment (stwiname) [inline Cronos/cronos-evm-starter-via-rpc/src/mappings/mappingHandlers.ts:26]: According to the abi these should be `wad` `src` dist`. The type at the top is wrong 

https://github.com/subquery/cosmos-subql-starter/blob/main/Cronos/cronos-evm-starter-via-rpc/erc20.abi.json#L50

File: Cronos/cronos-evm-starter-via-rpc/src/mappings/mappingHandlers.ts [hunk 2]
```diff
@@ -38,8 +37,8 @@ export async function handleEthermintEvmCall(
   const approval = Approval.create({
     id: call.hash,
     owner: call.from,
-    value: call.args._value.toBigInt(),
-    spender: call.args._spender,
+    value: call.args[1].toBigInt(),
```
Comment (stwiname) [inline Cronos/cronos-evm-starter-via-rpc/src/mappings/mappingHandlers.ts:40]: Same deal here https://github.com/subquery/cosmos-subql-starter/blob/main/Cronos/cronos-evm-starter-via-rpc/erc20.abi.json#L110

I'm not sure where this abi came from

**Pull request merged**